lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -172,6 +172,8 @@ def send_message(request, params = nil)
         json = message.to_json
 
         @stdin.write("Content-Length: #{json.length}\r\n\r\n", json)
+      rescue Errno::EPIPE
+        # The server connection died
```

Comment:
I didn't do it because it will end up printing too much. Once the server died, every single attempt to communicate with it will fail and print to stderr.

Considering that we don't have information about what caused the server to crash, it will be spamming the user with no actionable steps to take.

Reply:
I feel a simple message like `"Ruby LSP Rails server crashed and didn't respond"` will still help debugging a bit as we'll be sure the chain broke at this exact point. But IMO not a blocker, can add it later if proven needed too.

Reply:
My hypothesis is that if the server crashed we will already have printed something. I'll ship as is and we can add the print if we find it difficult to debug without it.